src/kolmogorov_solve.jl

```diff
@@ -0,0 +1,51 @@
+struct NNKolmogorov{C,O} <: NeuralNetDiffEqAlgorithm
+    chain::C
+    opt::O
+   
+end
+NNKolmogorov(chain  ; opt=Flux.ADAM(0.1)) = NNKolmogorov(chain , opt)
+ 
+function DiffEqBase.solve(
+    prob::KolmogorovPDEProblem,
+    alg::NNKolmogorov;
+    abstol = 1f-6,
+    verbose = false,
+    maxiters = 300,
+    save_everystep = false,
+    dt,
+    )
+
+    tspan = prob.tspan
+    phi = prob.phi
+    xspan = prob.xspan
+    sigma = prob.sigma
+    μ = prob.μ
+    d = prob.d
+    ts = tspan[1]:dt:tspan[2]
+    N = size(ts)
+    T = tspan[2]
+    #hidden layer
+    chain  = alg.chain
+    opt    = alg.opt
+
+    ps     = Flux.params(chain)
+    xi     = rand(Uniform(xspan[1] , xspan[2]), d , N[1] )
+    #Finding Solution to the SDE having initial condition xi. Y = Phi(S(X , T))
+    sdeproblem = SDEProblem(μ,sigma,xi,tspan)
+    sol = solve(sdeproblem,EM(),dt=dt , saveat = T)
+    x_sde = sol[2]
```

Comment:
why the second?

Reply:
The solution is saved at 0 and T, thereby taking the solution evaluated at T

Reply:
Then do save_everystep=false and sol[end]. Just a more robust style

Reply:
Seems like it would be nice to extend this to be time dependent though. A follow up making the algorithm time dependent and higher dimensional would be good.